src/main/java/org/opensrp/service/PractitionerService.java

```diff
@@ -117,4 +117,9 @@ public Practitioner getPractionerByUsername(String username) {
 
 		return getPractitionerRepository().getPractitionersByOrgId(organization.getId());
 	}
+
+	public Long getPractitionerIdByIdentifier(String identifier) {
```

Comment:
isn't this a duplicate of ` practitionerService.getPractitionerIdByIdentifier`

Reply:
There is a method which returns a Practitioner object, but I could not find a method which returns Id explicitly, therefore exposed a new method on the service layer **getPractitionerIdByIdentifier(String identifier)**

Reply:
maybe just return the id from the mentioned service method instead of using repository

Reply:
Noted